network:
  # names of the robots have to match hostnames described in /etc/hosts
  robot_names: [uav60, uav61, uav62, uav63, uav64, uav65, uav66, uav67, uav68, uav69, uav70, uav71]
